XExtensionItem/XExtensionItemParameters.m

```diff
@@ -13,6 +13,24 @@ @implementation XExtensionItemParameters
 
 #pragma mark - Initialization
 
+- (instancetype)initWithBlock:(void (^)(XExtensionItemMutableParameters *))initializationBlock {
+    NSParameterAssert(initializationBlock);
+    
+    XExtensionItemMutableParameters *parameters = [[XExtensionItemMutableParameters alloc] init];
+    
+    initializationBlock(parameters);
```

Comment:
Even though we parameter assert, I still think we should nil check this so we don’t crash the app that includes XExtensionItem in the case of developer error.